This patch updates text displayed in the about box to tell the user where they
can find a copy of the GPL on their system, since we don't install upstream's
LICENSE.TXT file.
--- a/about.c
+++ b/about.c
@@ -321,8 +321,8 @@
 
 static const char* 
 GBLlicense = 
-"ISO Master and bkisofs are distributed under the GNU General Public Licence\n"
-"version 2, please see LICENCE.TXT for the complete text\n";
+"ISO Master and bkisofs are distributed under the GNU General Public Licence version 2,\n"
+"on Debian systems please see /usr/share/common-licenses/GPL-2 for the complete text\n";
 
 static const char*
 GBLhelp = 
